src/init.cpp

```diff
@@ -517,6 +517,7 @@ void SetupServerArgs()
 
     gArgs.AddArg("-rest", strprintf("Accept public REST requests (default: %u)", DEFAULT_REST_ENABLE), ArgsManager::ALLOW_ANY, OptionsCategory::RPC);
     gArgs.AddArg("-rpcallowip=<ip>", "Allow JSON-RPC connections from specified source. Valid for <ip> are a single IP (e.g. 1.2.3.4), a network/netmask (e.g. 1.2.3.4/255.255.255.0) or a network/CIDR (e.g. 1.2.3.4/24). This option can be specified multiple times", ArgsManager::ALLOW_ANY, OptionsCategory::RPC);
+    gArgs.AddArg("-rpcallowstop", "Allow stop over a remote RPC client", ArgsManager::ALLOW_ANY, OptionsCategory::RPC);
```

Comment:
Suggest: "Accept shutdown (stop) commands from remote RPC clients."

Reply:
I also thought about this but I want that it is clear that it is referring to the "stop" command.